File: app/src/main/java/com/dynamic/dobler/elephantmath/database/converter/DateConverter.java
package com.dynamic.dobler.elephantmath.database.converter;

import java.text.SimpleDateFormat;
import java.util.Date;

public class DateConverter {

    public static String toNormalDate(Date date) {
        SimpleDateFormat format = new SimpleDateFormat("dd/MM/yyyy HH:mm");
        return format.format(date);
    }
}

File: app/src/main/java/com/dynamic/dobler/elephantmath/database/converter/DateConverter.kt
package com.dynamic.dobler.elephantmath.database.converter

import java.text.SimpleDateFormat
import java.util.*

object DateConverter {
    @JvmStatic
    fun toNormalDate(date: Date?): String {
        val format = SimpleDateFormat("dd/MM/yyyy HH:mm")
        return format.format(date)
    }
}